README.md

```diff
@@ -19,6 +19,11 @@ If that doesn't work:
  * [re-try with `LIBSSH2_SYS_USE_PKG_CONFIG=whatever`](https://github.com/nabijaczleweli/cargo-update/issues/129#issuecomment-599269219),
  * [install OpenSSL via `brew`, and re-try with `LDFLAGS="-L/usr/local/opt/openssl@1.1/lib" CPPFLAGS="-I/usr/local/opt/openssl@1.1/include"`](https://github.com/nabijaczleweli/cargo-update/issues/123),
  * [verify that you don't `openssl` installed twice via `brew`](https://github.com/nabijaczleweli/cargo-update/issues/121#issuecomment-570673813),
+ * [consult the `openssl` crate's instructions](https://github.com/sfackler/rust-openssl#macos),
+
+Starting with the next version you can enable the `vendored-openssl` feature to have libssh2 built against a statically built version of openssl as [described
```

Comment:
Next version of what?

Reply:
Oh I stole the README text from [libssh2](https://github.com/alexcrichton/ssh2-rs) and it listed the starting version so I figured best to say "next" in the README until the next release and then you could (or I could with another PR) fill in the version number on the next release of this package. If it is accepted of course :)

Reply:
Or can change it to anything you rather have it say